src/guides/v2.3/howdoi/checkout/checkout_new_field.md

```diff
@@ -102,7 +102,13 @@ define([
                 shippingAddress['extension_attributes'] = {};
             }
 
-            shippingAddress['extension_attributes']['custom_field'] = shippingAddress.customAttributes['custom_field'];
+            var attribute = shippingAddress.customAttributes.find(
+                    function (element) {
```

Comment:
Indenting is not standard format. Lines 106/108 are indented 8 spaces - should be 4. Line 107 should be indented 4 from 106